src/tools/tools/Logger.fs

```diff
@@ -297,6 +311,16 @@ type internal DefaultLogger () =
         member this.Log ((logId : string, logLevel : LogLevel, message : string)) =
             // DefaultLogger does not support "log id" yet
             EmitLogEntry(logLevel, message)
+        
+        member this.Log ((logId : string, jobID: Guid, logLevel : LogLevel, message : string)) =
```

Comment:
The "Logger" class below should have a non-functional flavor "Log" function that takes JobID as parameter